src/backend/core/core/mixins.py

```diff
@@ -51,6 +56,8 @@ def to_representation(self, instance):
         inst_rep = super().to_representation(instance)
         # request = self.context.get('request')
         # lang_code = request.META.get('HTTP_ACCEPT_LANGUAGE', None)
+        # request = self.context.get("request")
+        # lang_code = get_language_from_request(request)
```

Comment:
IMO, there could be conflicts with my changes
so we just need to make sure everything is working after the merge

Reply:
Yeah, let's merge your changes `get_language_from_request` and we will see. I think I had some issues with it so hence why I change it. But we will see :)